awx/main/migrations/0068_v350_resource_profiling.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
This needs to be `0069`, I think @rooftopcellist got `0068` yesterday.

Reply:
Rebased and bumped version number